src/test/kotlin/com/charleskorn/kaml/YamlNoDefaults.kt

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
Could you please move this into the main test class? I know it's huge (and I will split it at some point), but at the moment that's where all the other tests live and it seems weird to just have these tests split out.